public/lib/general.js

```diff
@@ -1,121 +1,136 @@
-(function() {
-	"use strict";
-	
-	var articlePath = window.location.protocol + '//' + window.location.host + window.location.pathname;
+'use strict';
 
-	var pluginURL = nodeBBURL + '/plugins/nodebb-plugin-blog-comments',
-		savedText, nodebbDiv, contentDiv, commentsDiv, commentsCounter, commentsAuthor, commentsCategory;
+/* eslint-disable no-undef */
+(function () {
+	const articlePath = `${window.location.protocol}//${window.location.host}${window.location.pathname}`;
 
-	var stylesheet = document.createElement("link");
-	stylesheet.setAttribute("rel", "stylesheet");
-	stylesheet.setAttribute("type", "text/css");
-	stylesheet.setAttribute("href", pluginURL + '/css/comments.css');
+	const pluginURL = `${nodeBBURL}/plugins/nodebb-plugin-blog-comments`;
+	let savedText; let contentDiv; let commentsDiv; let commentsCounter; let commentsAuthor; let
+		commentsCategory;
 
-	document.getElementsByTagName("head")[0].appendChild(stylesheet);
+	const stylesheet = document.createElement('link');
+	stylesheet.setAttribute('rel', 'stylesheet');
+	stylesheet.setAttribute('type', 'text/css');
+	stylesheet.setAttribute('href', `${pluginURL}/css/comments.css`);
+
+	document.getElementsByTagName('head')[0].appendChild(stylesheet);
 	document.getElementById('nodebb-comments').insertAdjacentHTML('beforebegin', '<div id="nodebb"></div>');
-	nodebbDiv = document.getElementById('nodebb');
+	const nodebbDiv = document.getElementById('nodebb');
 
 	function newXHR() {
```

Comment:
Can probably switch to `fetch` here, but for now you can remove the extremely old ActiveX stuff and use XMLHttpRequest directly.